spec/files/example_project_combine_GHE_2.json

```diff
@@ -760,6 +760,7 @@
         "geometryType": "Rectangle",
         "name": "New District System_1",
         "type": "District System",
+        "districtSystemType":"Ground Heat Exchanger",
```

Comment:
Has this been added to the schema already?

Reply:
@vtnate , yes https://github.com/urbanopt/urbanopt-geojson-gem/blob/develop/lib/urbanopt/geojson/schema/district_system_properties.json#L88